src/app/forms/custom-sections/last-applicant-section/last-applicant-section-edit/last-applicant-section-edit.component.ts

```diff
@@ -63,31 +63,31 @@ export class LastApplicantSectionEditComponent implements OnInit, OnDestroy {
 	get defaultFields() {
 		return {
 			techRecord_applicantDetails_name: this.fb.control(null, [
-				this.commonValidators.max(150, 'Name must be less than or equal to 150'),
+				this.commonValidators.maxLength(150, 'Name or company must be less than or equal to 150 characters'),
 			]),
 			techRecord_applicantDetails_address1: this.fb.control(null, [
-				this.commonValidators.max(60, 'Address line 1 must be less than or equal to 60'),
+				this.commonValidators.maxLength(60, 'Address line 1 must be less than or equal to 60 characters'),
 			]),
 			techRecord_applicantDetails_address2: this.fb.control(null, [
-				this.commonValidators.max(60, 'Address line 2 must be less than or equal to 60'),
+				this.commonValidators.maxLength(60, 'Address line 2 must be less than or equal to 60 characters'),
 			]),
 			techRecord_applicantDetails_postTown: this.fb.control(null, [
-				this.commonValidators.max(60, 'Town or City must be less than or equal to 60'),
+				this.commonValidators.maxLength(60, 'Town or City must be less than or equal to 60 characters'),
 			]),
 			techRecord_applicantDetails_address3: this.fb.control(null, [
-				this.commonValidators.max(60, 'County must be less than or equal to 60'),
+				this.commonValidators.maxLength(60, 'County must be less than or equal to 60 characters'),
 			]),
 			techRecord_applicantDetails_postCode: this.fb.control(null, [
-				this.commonValidators.max(12, 'Postcode must be less than or equal to 12'),
+				this.commonValidators.maxLength(12, 'Postcode must be less than or equal to 12 characters'),
 			]),
 			techRecord_applicantDetails_telephoneNumber: this.fb.control(null, [
-				this.commonValidators.max(25, 'Telephone number must be less than or equal to 25'),
+				this.commonValidators.maxLength(25, 'Telephone number must be less than or equal to 25 characters'),
 			]),
 			techRecord_applicantDetails_emailAddress: this.fb.control(null, [
-				this.commonValidators.max(255, 'Email address must be less than or equal to 255'),
+				this.commonValidators.maxLength(255, 'Email address must be less than or equal to 255 characters'),
 				this.commonValidators.pattern(
 					"^[\\w\\-\\.\\+']+@([\\w-]+\\.)+[\\w-]{2,}$",
-					'Enter an email address in the correct format, like name@example.com'
+					'Email address Enter an email address in the correct format, like name@example.com'
```

Comment:
Bit of an odd message here - is that what develop is?

Reply:
yeah it is

Reply:
i had to change this for the regression pack as it was causing failures